pkg/resources/kubernetes-dashboard/deployment.go

```diff
@@ -188,9 +188,12 @@ func getDashboardVersion(clusterVersion semver.Semver) (string, error) {
 	case "1.21":
 		return "v2.4.0", nil
 	case "1.22":
+		// nolint:goconst
```

Comment:
I did not want to make it a const because it is one of the TODOs that you mentioned in a comment above. So it’s going to go away anyway and then the constant doesn’t make much sense anymore.